server/routes/shared_api/choropleth.py

```diff
@@ -238,7 +238,7 @@ def geojson():
     place_dcid, place_type = get_choropleth_display_level(place_dcid)
   place_name_prop = request.args.get("placeNameProp")
   geojson_prop = request.args.get("geoJsonProp",
-                                  CHOROPLETH_DEFAULT_GEOJSON_PROP)
+                                  current_app.config["GEO_JSON_PROP"])
```

Comment:
To confirm, when web-components sets geoJsonProp that will override the backend config?  Add a comment to that effect maybe?

Reply:
correct, if the client calls this endpoint with a geoJsonProp, that value will be used. Added a comment.